notebooks/visit_data.py

```diff
@@ -0,0 +1,255 @@
+# can we predict the next site to be visited from the current site?
```

Comment:
Think this belongs in the link-analysis branch which we already merged into `main`. Will revisit later.